run/grpc-ping/e2e_test.go

```diff
@@ -0,0 +1,116 @@
+package main_test
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/cloudrunci"
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/metadata"
+
+	pb "github.com/GoogleCloudPlatform/golang-samples/run/grpc-ping/pkg/api/v1"
+)
+
+// TestGRPCPingService is an end-to-end test that confirms ping responses work as expected with authentication.
+// Test Cases:
+// - Authenticated request from client to ping service
+// - Authenticated request from client to ping service w/ recursive upstream request
+func TestGRPCPingService(t *testing.T) {
+	tc := testutil.EndToEndTest(t)
+
+	// Prepare the container image for both services.
+	pingService := cloudrunci.NewService("grpc-ping", tc.ProjectID)
+	//pingService.Build()
```

Comment:
Delete or uncomment.

Reply:
Ack. This would cause the E2E tests to fail.